PalasoUIWindowsForms/ImageToolbox/ArtOfReadingChooser.cs

```diff
@@ -190,24 +198,25 @@ private void ArtOfReadingChooser_Load(object sender, EventArgs e)
 				//label1.Visible = _searchTermsBox.Visible = _searchButton.Visible = _thumbnailViewer.Visible = false;
 				_messageLabel.Visible = true;
 				_messageLabel.Font = new Font(SystemFonts.DialogFont.FontFamily, 10);
-				_messageLabel.Text = @"This computer doesn't appear to have the 'International Illustrations: the Art Of Reading' gallery installed yet.";
+				_messageLabel.BackColor = Color.White;
```

Comment:
Why are we having to mess with BackColor? This would normally be in designer...I don't see that it is ever set to anything but white.